_posts/2015-02-17-aerogear-webpush-alpha1.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: AeroGear WebPush
```

Comment:
the value of the title is used on the "Hey listen! ...." section of the side.

I'd suggest to use a different title, like -> _AeroGear WebPush alpha.1 is out_.

The generated content on the site would be "Hey listen! AeroGear WebPush alpha.1 is out"